docs/source/readme_links.md

```diff
@@ -0,0 +1,9 @@
+
+# README Links
```

Comment:
Are we worried about this getting out of date?

Reply:
I have removed these links